site/src/pug/includes/body.pug

```diff
@@ -38,7 +40,11 @@ include ./fork.pug
             each swag in swagList
                 - difficulty = swag.difficulty + ' difficulty';
                 - difficultyTitle = 'Difficulty: ' + swag.difficulty;
-                div.item
+                div(
+                    class="item visible " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```

Comment:
Is there a reason why you add the space in map's literal?

```suggestion
                    class="item visible " + swag.difficulty + swag.tags.map(t => `tag-${t}`).join(' ')
```

Reply:
If I remember correctly it's because `join` does `a b c d` while space does ` a b c d` which is what's needed here

Reply:
Sure!